### ISSUE TITLE
Numeric coloring model initialisation

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#810]
I am currently refactoring the initialization of MoBIE and I would like to avoid the loading of tables during initialization (because this can take quite some time, e.g. in the HTM case).

Now, for the serialization of the coloring, we currently have:

```
	protected String lut;
	protected String colorByColumn;
	protected Double[] valueLimits;
```

I think I can avoid loading of tables for initialization of the coloring model except for one case: 

(a) `colorByColumn`  is set.
(b) `lut` is a _numerical_ LUT.
(c) `valueLimits` is _not_ set.

Because here we currently need to fully load the respective column in order to find the min and max value for the automated initialization of the `valueLimits`.

@constantinpape Would it be OK to say in the spec that in the above case (a)&(b) `valueLimits` MUST be set?

**Comment by constantinpape** [comment]
> @constantinpape Would it be OK to say in the spec that in the above case (a)&(b) `valueLimits` MUST be set?

Yes, that's ok. It's hard to express this statically in the spec, since expressing dependencies between different fields is not very easy in jsonschema (it's possible, but makes everything very hard to read). So for now I suggest we just write it in the spec, and I can add additional checks in the python validation code where this is much easier to check.

Which brings me to the question: which `lut`s do we currently have that are numeric?

**Comment by tischi** [comment]
Numeric Luts:
- `blueWhiteRed`
- `viridis`
